docs/install/vm-install/ubuntu.md

```
@@ -80,47 +98,48 @@ Download the release package from the GitHub FLEX [Releases](https://github.com/
-Verify integrity of the downloaded package by verifying published `sha256sum`.
-
-  * Go to the [Flex Project Releases page](https://github.com/gluufederation/flex/releases) and copy the `sha256sum` value for the `flex_replace-flex-version-stable.ubuntu22.04_amd64.deb` file:
-  * Replace `paste-release-sha256sum` in the command below with the actual checksum you copied from the release page, and run the following command:
-      ```bash title="Command"
-      echo 'paste-release-sha256sum flex_replace-flex-version-stable.ubuntu22.04_amd64.deb ' | sed 's/^sha256://' > flex_replace-flex-version-stable.ubuntu22.04_amd64.deb .sha256sum && sha256sum -c flex_replace-flex-version-stable.ubuntu22.04_amd64.deb.sha256sum
-      ```
-  * Output similar to below should confirm the integrity of the downloaded package.
     ```bash title="Command"
-    flex_replace-flex-version-stable.ubuntu22.04_amd64.deb: OK
+    cd /tmp
     ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)